CI/package-mudlet-for-windows.sh

```
@@ -191,6 +159,63 @@ mkdir ./brimworks
+  | /usr/bin/grep 'bin' \
+  | /usr/bin/cut -d '>' -f2 \
+  | /usr/bin/cut -d '(' -f1 \
+  | /usr/bin/sort -u)
+
+# echo ""
+# echo "  In summary, the needed libraries are:"
+# echo "${NEEDED_LIBS[@]}"
+
+echo ""
+echo "Copying identified libraries from Mudlet executable and plugins..."
+for LIB in ${NEEDED_LIBS[@]}; do
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)